claimManagement/src/main/java/org/openimis/imisclaims/network/dto/ActivityDefinitionDto.java

```diff
@@ -0,0 +1,127 @@
+package org.openimis.imisclaims.network.dto;
+
+import androidx.annotation.NonNull;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+
+public class ActivityDefinitionDto {
+
+    private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.US);
+
+    @NonNull
+    public static List<ActivityDefinitionDto> fromJson(@NonNull JSONObject jsonObject) throws JSONException, ParseException {
```

Comment:
Since Java is statically typed and these are clearly utility methods, I don't find it confusing, and making the name longer would not make the code cleaner, IMHO.

Reply:
characters are not that expensive that you have to spare them ;).

anyway fine by me.